content/en/docs/tutorials/stateless-application/guestbook.md

```diff
@@ -39,24 +35,27 @@ This tutorial shows you how to build and deploy a simple, multi-tier web applica
 
 <!-- lessoncontent -->
 
-## Start up the Redis Master
+## Start up the Mongo Database
 
-The guestbook application uses Redis to store its data. It writes its data to a Redis master instance and reads data from multiple Redis slave instances.
+The guestbook application uses MongoDB to store its data.
 
-### Creating the Redis Master Deployment
+### Creating the Mongo Deployment
 
-The manifest file, included below, specifies a Deployment controller that runs a single replica Redis master Pod.
+The manifest file, included below, specifies a Deployment controller that runs a single replica MongoDB Pod.
 
-{{< codenew file="application/guestbook/redis-master-deployment.yaml" >}}
+{{< codenew file="application/guestbook/mongo-deployment.yaml" >}}
 
 1. Launch a terminal window in the directory you downloaded the manifest files.
-1. Apply the Redis Master Deployment from the `redis-master-deployment.yaml` file:
+1. Apply the MongoDB Deployment from the `mongo-deployment.yaml` file:
 
       ```shell
-      kubectl apply -f https://k8s.io/examples/application/guestbook/redis-master-deployment.yaml
+      kubectl apply -f https://k8s.io/examples/application/guestbook/mongo-deployment.yaml
       ```
+<!---
+kubectl apply -f ./content/en/examples/application/guestbook/mongo-deployment.yaml
+-->
```

Comment:
I'd omit that.

Reply:
It's a comment that does not show up in the rendered markdown, but instead lets a person reviewing/editing the document test the commands via path ( as obviously they'll be different via the url )